### PR TITLE
Feature/minified crops

### PR DESCRIPTION
**Pull request by Adamws33**
updated crops data call to use minified crops call.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,20.75.190.128,:,80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSidebar/CropSidebar.js [hunk 1]
```diff
@@ -146,20 +146,12 @@ const CropSidebar = ({
...
+        if (crop.attributes.filter((att) => att.label === key).length > 0) {
+          // if there is not an intersection, match = false
+          if (!crop.attributes.filter((att) => att.label === key)[0]?.values.some((item) => vals.includes(item))) {
+            match = false;
           }
-        });
+        }
       });
 
       cd[n].inactive = (!match)
```
Comment (YazidZidane) [inline src/pages/CropSidebar/CropSidebar.js:157]: I got an error on this line when I'm testing in Kentucky:
`CropSidebar.js:157 Uncaught TypeError: Cannot read properties of undefined (reading 'Soil Conditions')
    at CropSidebar.js:157:1`

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
it looks like filtering is broken now though. going to nc and changing any of the filters doesn't filter down the crops

**Comment by mikahpinegar** [comment]
the same issue exists for all the councils, filtering isn't working anymore

**Comment by mikahpinegar** [comment]
comparison view is no longer showing the icons for ratings, instead showing text
comp

**Comment by mikahpinegar** [comment]
comparison view is no longer showing dollar signs icons for price
price

**Comment by mikahpinegar** [comment]
can you add back the styling for key traits where units are grayed out?

**Comment by mikahpinegar** [comment]
whenever i select any goal for nc and then press next to view the selector i get this error
```
Cannot read properties of undefined (reading 'values')
TypeError: Cannot read properties of undefined (reading 'values')
    at http://localhost:3000/static/js/bundle.js:5645:127
    at Array.map (<anonymous>)
    at http://localhost:3000/static/js/bundle.js:5627:74
    at Array.map (<anonymous>)
    at RenderCrops (http://localhost:3000/static/js/bundle.js:5498:86)
    at renderWithHooks (http://localhost:3000/static/js/bundle.js:144257:22)
    at mountIndeterminateComponent (http://localhost:3000/static/js/bundle.js:147543:17)
    at beginWork (http://localhost:3000/static/js/bundle.js:148839:20)
    at HTMLUnknownElement.callCallback (http://localhost:3000/static/js/bundle.js:133849:18)
    at Object.invokeGuardedCallbackDev (http://localhost:3000/static/js/bundle.js:133893:20)
```

**Comment by mikahpinegar** [comment]
sorting by goals isn't working properly. it does reorder the crops but not correctly

**Comment by mikahpinegar** [comment]
> > sorting by goals isn't working properly. it does reorder the crops but not correctly
> 
> the goals stuff wont work till the data is updated from trevor. some crops are missing goals i think he said he was changing them to null but it was still in progress.

it should still sort the crops that do have ratings. for nc all the crops have ratings for the goals i'm looking at and it's still not sorting correctly

**Comment by mikahpinegar** [comment]
@trevorjpuckett i forget what we discussed about the null returns for missing data. is that already happening or is it still in development?

**Comment by Adamws33** [comment]
table view not sorting properly on filter select 

image

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by trevorjpuckett** [comment]
@mikahpinegar that should have been in my latest PR for the API. I believed we merged that in last week.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
default zone is not working. picking nc in landing and then navigating to browse cover crops has no zone selected by defaut

**Comment by mikahpinegar** [comment]
list view and calendar view buttons are backwards in the sidebar. when on list view it says on calendar view and vice versa

**Comment by mikahpinegar** [comment]
default sorting by average goal rating isn't working. crops are sorted alphabetically on first render

**Comment by mikahpinegar** [comment]
filtering on the list view no longer grays crops out, it removes them from the crop list. this is incorrect behavior

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,20.75.190.128,:,80,'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
crops are sorted in reverse average goal rating on first render

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
this pr also has conflicts now

**Comment by mikahpinegar** [comment]
comparison view and list view buttons are flipped in view my list. comparison view should show the table. also there are still conflicts

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
search isn't working on the list view

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.75.190.128:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Pull request merged**